# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/humantime/Cargo.toml
# Commit: a651d4323066
# Full Hash: a651d43230664837c308663382ccc3bcd7298dc2
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r be008ddb2418 -r a651d4323066 third_party/rust/humantime/Cargo.toml
--- a/third_party/rust/humantime/Cargo.toml	Fri Jan 27 21:25:50 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,37 +0,0 @@
-# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
-#
-# When uploading crates to the registry Cargo will automatically
-# "normalize" Cargo.toml files for maximal compatibility
-# with all versions of Cargo and also rewrite `path` dependencies
-# to registry (e.g., crates.io) dependencies
-#
-# If you believe there's an error in this file please file an
-# issue against the rust-lang/cargo repository. If you're
-# editing this file be aware that the upstream Cargo.toml
-# will likely look very different (and much more reasonable)
-
-[package]
-edition = "2018"
-name = "humantime"
-version = "2.1.0"
-authors = ["Paul Colomiets <paul@colomiets.name>"]
-description = "    A parser and formatter for std::time::{Duration, SystemTime}\n"
-homepage = "https://github.com/tailhook/humantime"
-documentation = "https://docs.rs/humantime"
-readme = "README.md"
-keywords = ["time", "human", "human-friendly", "parser", "duration"]
-categories = ["date-and-time"]
-license = "MIT/Apache-2.0"
-repository = "https://github.com/tailhook/humantime"
-
-[lib]
-name = "humantime"
-path = "src/lib.rs"
-[dev-dependencies.chrono]
-version = "0.4"
-
-[dev-dependencies.rand]
-version = "0.6"
-
-[dev-dependencies.time]
-version = "0.1"